### DISCUSSION TITLE
is `xtream_info_cache` deprecated.?

### DISCUSSION THREAD

**Discussion by aniel300** [euzu/tuliprox#110] [General]
is `xtream_info_cache` deprecated.? there is some discrepancy between the readme and the wiki.

**Comment by euzu** [comment]
yes it is deprecated with the release 2.0.6.
I will update the wiki.

**Reply by aniel300** [reply]
might i ask why it it got deprecated, it seems like a useful feature

**Reply by euzu** [reply]
becauae it cant be optional anymore. This is technically needed. The info has to be processed and saved.

**Reply by aniel300** [reply]
ahh so is enabled by default? i can live with that, thanks.

**Reply by euzu** [reply]
No it cant be disabled. If you could you could not watch vod and series.

**Reply by aniel300** [reply]
yes that is what am saying. it is enabled by default which am fine with.
